src/lib/components/LoginModal.svelte

```diff
@@ -22,11 +22,18 @@
 		<p class="text-balance text-lg font-semibold leading-snug text-gray-800">
 			{envPublic.PUBLIC_APP_DESCRIPTION}
 		</p>
-		<p class="text-balance rounded-xl border bg-white/80 p-2 text-base text-gray-800">
-			You have reached the guest message limit, <strong class="font-semibold"
-				>Sign In with a free Hugging Face account</strong
-			> to continue using HuggingChat.
-		</p>
+			{#if envPublic.MESSAGES_BEFORE_LOGIN > 0}
```

Comment:
Thanks for the review. In the official documentation, under the section on local installation, it states that MESSAGES_BEFORE_LOGIN can be set to 0 to force immediate login. Otherwise, this variable should be set to a numeric value that represents the number of messages allowed for guests. In this scenario, guests can exist, and a custom guest message can be displayed.

While testing locally, I observed the following:
- Setting MESSAGES_BEFORE_LOGIN=0: I was able to write a prompt, but got an error message, "Something went wrong" upon sending. This issue could be resolved in another PR to display the login screen directly instead of showing the chat interface, similar to other chat UI projects.
- Setting MESSAGES_BEFORE_LOGIN to a value greater than 0: I was allowed to send the specified number of messages before being prompted to log in, and the default guest message was displayed. 
- Another solution could be introducing a new environment variable, PUBLIC_APP_GUEST, similar to PUBLIC_APP_DISCLAIMER. If PUBLIC_APP_GUEST is set to 1, a custom guest message would be allowed, otherwise, a default message would be used.

Reply:
Oh, you are right! Sorry, never worked with Svelte before :D 

The logic was, that by setting the MESSAGES_BEFORE_LOGIN to a numeric value, we are sure that there will be a guest and therefore we might need to display a custom message. 

We can remove the line entirely or introduce a new environment variable, PUBLIC_APP_GUEST, similar to PUBLIC_APP_DISCLAIMER. If PUBLIC_APP_GUEST is set to 1, a custom guest message would be allowed, otherwise, a default message would be used.

Reply:
Oh, you are right! Sorry, never worked with Svelte before :D

The logic was, that by setting the MESSAGES_BEFORE_LOGIN to a numeric value, we are sure that there will be a guest and therefore we might need to display a custom message.

We can remove the line entirely as you suggested below or introduce a new environment variable, PUBLIC_APP_GUEST, similar to PUBLIC_APP_DISCLAIMER. If PUBLIC_APP_GUEST is set to 1, a custom guest message would be allowed, otherwise, a default message would be used.